packages/ra-core/src/i18n/Translate.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useTranslate } from './useTranslate';
+
+export const Translate = ({
+    i18nKey,
+    args,
+    children = 'no translation',
```

Comment:
I think it should default to empty striong if no translation and no default value.